src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,138 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final User NULL_USER = null;
+    private static final Integer NULL_AGE = null;
+    private static final String NULL_LOGIN = null;
+    private static final String NULL_PASSWORD = null;
+    private static final String VALID_LOGIN1 = "Andrew";
+    private static final String VALID_LOGIN2 = "Penelopa";
+    private static final String VALID_PWD = "1234567";
+    private static final int VALID_AGE = 20;
+    private static final String INVALID_LOGIN = "Andr";
+    private static final String INVALID_PWD = "12345";
+    private static final int INVALID_AGE = 17;
+    private static final int MIN_AGE = 18;
+    private static final int MIN_LENGTH = 6;
+    private static final User INVALID_LOGIN_USER = new User(INVALID_LOGIN, VALID_PWD, VALID_AGE);
```

Comment:
Please do not hardcode users, it is better to create them in test methods using constuctor/factory method/builder